h2o-algos/src/test/java/hex/generic/GenericModelTest.java

```diff
@@ -839,7 +839,7 @@ private void testJavaScoringCoxPH(Frame trainingFrame, Frame testFrame, String[]
 
         final Frame originalModelPredictions = originalModel.score(testFrame);
         Scope.track(originalModelPredictions);
-        assertTrue(TestUtil.compareFrames(genericModelPredictions, originalModelPredictions, 0.000001));
+        assertTrue(TestUtil.compareFrames(genericModelPredictions, originalModelPredictions, 0.000001, 0.00001));
```

Comment:
the order of args should be `originalModelPredictions, genericModelPredictions` since it is exp, actual

Reply:
Updated